### DISCUSSION TITLE
#30DaysOfSWA: Introduce Yourself!

### DISCUSSION THREAD

**Discussion by nitya** [staticwebdev/30DaysOfSWA#2] [2. #30DaysOfSWA]
Let's use this topic to introduce ourselves as we kickstart the #30DaysOfSWA journey. Consider sharing something like the following (all prompts optional!)

 * Name and Pronouns - so we can say 👋🏽
 * Location - if you want to connect with people in your area.
 * Goals - what you hope to get out of exploring #30DaysOfSWA
 * Interests - what technologies you are working with (or interested in skilling up on)
 * Brag time - share something you are proud of yourself for doing. Does not have to be tech related.
 
---

 
![image](https://user-images.githubusercontent.com/279083/166150467-4479bfc2-151e-49fd-97dd-b46a4a3db3f0.png)

**Comment by nitya** [comment]
A quick introduction to me to kick things off:
 * I'm Nitya 👋🏽 - you can find me on [@nitya](https://twitter.com/nitya) and [dev.to](https://dev.to/nitya)
 * I'm based in the New York region of the US - if you're from this region, say hello! 🗽🇺🇸
 * I'm the core creator and maintainer for the [#30DaysOfSWA](https://aka.ms/30DaysOfSWA) project - it's a labor of love ❤️
 * I'm currently in the JavaScript Advocacy team at Microsoft - love all things web, mobile, cloud and AI! 👩🏽‍💻
 * I love Visual Storytelling [@sketchthedocs](https://twitter.com/sketchthedocs) - Satya Nadella [shared my work](https://twitter.com/nitya/status/1456640327943696389) and [180K people saw it](https://www.linkedin.com/feed/update/urn:li:activity:6861658636184240129/?updateEntityUrn=urn%3Ali%3Afs_feedUpdate%3A%28V2%2Curn%3Ali%3Aactivity%3A6861658636184240129%29) 🎉
 

One last thing. I worked in ubiquitous computing research (for a decade), then discovered web development and community advocacy as a means for bringing ideas to life and giving back to others. **No matter your path to this point, you belong here in tech and I am glad to see you here!** Welcome 👏🏽

**Reply by shelDev** [reply]
@nitya - thank you for creating and maintaining the #30DaysOfSWA project!

- I am Michelle 👋🏽 - @shelDev and [@Shel_i_am](https://twitter.com/shel_i_am)
- I am based in central Wisconsin
- 23 years ago I was the first webmaster & administration for a federal corporate credit union in San Dimas, CA and I programmed their first corporate banking app, WAVE, using C++, vbScript and active server pages.👩🏽‍💻
- Since that time, I have done very little in tech, and instead created and developed 3 humans. ❤️❤️❤️
- I am a certified peer support specialist and I provide volunteer support for clients with mental health and/or substance abuse issues.
- I am looking forward to attempting a project during the SWA anniversary!

**Reply by nitya** [reply]
Hi @shelDev !! Welcome - I am so glad to see you here and thanks for leaving the note. 
I used to live in Chicagolands close to Kenosha, WI so nice to see someone from that part of the US!
If I can help make your SWA goal happen, do let me know - I think developing 3 humans is much harder than software (the bugs are harder to detect and fix!) so kudos to you for doing both!! Wish you all success in the journey!

**Comment by myaesubi** [comment]
Hi @nitya ,
I tried the content page both in Chrome, Edge and also in incognito browsing. it appears the styling gets corrupted. post first Kick off page.

![image](https://user-images.githubusercontent.com/177969/167208711-7e5ade53-3ddd-4291-9a68-4f44838eac46.png)

**Reply by nitya** [reply]
Hi @myaesubi - thanks for the update! Can you try using https://aka.ms/30DaysOfSWA directly?
The site was moved from GitHub Pages to Azure Static Web Apps and should be available there.
